content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -143,214 +90,82 @@ Once you have a Linux-based Kubernetes control-plane ("Master") node you are rea
     After a few minutes, you should see all the pods as running if the Flannel pod network was deployed.
 
     ```bash
-    kubectl get pods --all-namespaces
+    kubectl get pods -n kube-system
     ```
 
-    The output looks like as follows:
+    The output should include the Linux flannel DaemonSet as running:
 
     ```
     NAMESPACE     NAME                                      READY        STATUS    RESTARTS   AGE
-    kube-system   etcd-flannel-master                       1/1          Running   0          1m
-    kube-system   kube-apiserver-flannel-master             1/1          Running   0          1m
-    kube-system   kube-controller-manager-flannel-master    1/1          Running   0          1m
-    kube-system   kube-dns-86f4d74b45-hcx8x                 3/3          Running   0          12m
+    ...
     kube-system   kube-flannel-ds-54954                     1/1          Running   0          1m
-    kube-system   kube-proxy-Zjlxz                          1/1          Running   0          1m
-    kube-system   kube-scheduler-flannel-master             1/1          Running   0          1m
     ```
 
-    Verify that the Flannel DaemonSet has the NodeSelector applied.
-
-    ```bash
-    kubectl get ds -n kube-system
-    ```
+1. Add Windows Flannel and kube-proxy DaemonSets
 
-    The output looks like as follows. The NodeSelector `beta.kubernetes.io/os=linux` is applied.
+    Now we will add Windows-compatible versions of Flannel and kube-proxy. In order
```

Comment:
[Avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), if feasible.